src/actions/generate-fixtures.ts

```diff
@@ -179,15 +180,12 @@ const generateMessageSign = async (mnemonic: string) => {
     const message = "Hello, world!";
     const privateKey = await privateKeyFactory.fromMnemonic(mnemonic);
     const publicKey = await publicKeyFactory.fromMnemonic(mnemonic);
-    const signature = await signatureFactory.signRecoverable(
-        Buffer.from(keccak256(toHex(message)).slice(2), "hex"),
-        Buffer.from(privateKey, "hex"),
-    );
+    const signature = new SigningKey(`0x${privateKey}`).sign(hashMessage(message))
 
     await writeFixtureToFile("message-sign.json", {
         message,
         publicKey,
-        signature: signature.r + signature.s + signature.v.toString(16).padStart(2, "0"),
+        signature: signature.serialized
```

Comment:
This results in it ending with 27 or 28. 

Taken from my comment [here](https://github.com/ArdentHQ/typescript-crypto/pull/59/files#r2214483251):
Also, will it impact anything in ts-crypto if `0x` is removed from the start? it didn't used to exist and it saves handling it in tests on other SDKs. It's not a biggie, just that no other fixtures have `0x` in them (apart from addresses)